packages/investor-idle/src/idlecli.ts

```diff
@@ -0,0 +1,88 @@
+import { ethereum } from '@shapeshiftoss/chain-adapters'
+import { NativeAdapterArgs, NativeHDWallet } from '@shapeshiftoss/hdwallet-native'
+import * as unchained from '@shapeshiftoss/unchained-client'
+import dotenv from 'dotenv'
+
+import { bnOrZero } from './utils'
+import { IdleInvestor } from './IdleInvestor'
+
+dotenv.config()
+
+const { DEVICE_ID = 'device123', MNEMONIC } = process.env
+
+const getWallet = async (): Promise<NativeHDWallet> => {
+  if (!MNEMONIC) {
+    throw new Error('Cannot init native wallet without mnemonic')
+  }
+  const nativeAdapterArgs: NativeAdapterArgs = {
+    mnemonic: MNEMONIC,
+    deviceId: DEVICE_ID
+  }
+  const wallet = new NativeHDWallet(nativeAdapterArgs)
+  await wallet.initialize()
+
+  return wallet
+}
+
+const main = async (): Promise<void> => {
+  const wallet = await getWallet()
+  const chainAdapter = new ethereum.ChainAdapter({
+    providers: {
+      ws: new unchained.ws.Client<unchained.ethereum.Tx>('wss://dev-api.ethereum.shapeshift.com'),
+      http: new unchained.ethereum.V1Api(
+        new unchained.ethereum.Configuration({
+          basePath: 'https://dev-api.ethereum.shapeshift.com'
+        })
+      )
+    },
+    rpcUrl: 'https://mainnet.infura.io/v3/d734c7eebcdf400185d7eb67322a7e57'
+  })
+
+  const idleInvestor = new IdleInvestor({
+    providerUrl: 'https://daemon.ethereum.shapeshift.com',
+    dryRun: true,
+    chainAdapter
+  })
+
+  const address = '0x358dae76Bb42Be167dD5A64f95E0d537b024834e'
+  const usdcCaip19 = 'eip155:1/erc20:0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'.toLowerCase()
```

Comment:
Edit: Wrong with the current implementation so not expected for you to change it! 

~~We use the `AssetId` terminology, in this case`usdcAssetId` - also might sound nitpicky but standards matter: since the CAIP-19s are case sensitive, you might want to keep this version with the checksum casing and lowercase the consumption~~